runtime/karura/src/lib.rs

```diff
@@ -1403,6 +1405,29 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
 }
 
+parameter_types! {
+	pub const KSMCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::KSM);
+	pub const LKSMCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::LKSM);
+	pub const MinimumMintThreshold: Balance = 1_000_000_000;
+	pub RelaychainSovereignSubAccount: MultiLocation = MultiLocation::X2( Junction::Parent, Junction::AccountId32 {
+		network: RelayNetwork::get(),
+		id: ParachainInfo::get().into_account(),
```

Comment:
@xlc we may need a derive path to get the sovereign sub-account id, any suggestions?

Reply:
https://github.com/paritytech/substrate/blob/7960c9eba8c1a73e8792c200594544d65b1333bd/frame/utility/src/lib.rs#L310

this is public, so we should be able to call it to calculate the account